src/utils/hooks/useFetchLogger.ts

```diff
@@ -0,0 +1,18 @@
+import { FetchResult, Status } from '@nutgaard/use-fetch';
+import { useEffect } from 'react';
+import { loggEvent } from '../frontendLogger';
+import { usePrevious } from '../customHooks';
+
+export function useFetchLogger(data: FetchResult<any>, location: string, type?: string) {
```

Comment:
Kunne ikke denne vært noe alà:
```
function useFetchWithLog<TYPE>(
  url: string,
  options?: RequestInit,
  config: Config,
  type?: string
): FetchResult<TYPE> {
  const result = useFetch(url, options, config);
  const { status } = result;
  
  useEffect(() => {
    if (status === Status.PENDING) {
      loggEvent('Fetch', url, { type: type });
    } else if (status === Status.ERROR) {
      loggEvent('Fetch-Failed', url, { type: type });
    } else if (status === Status.RELOADING) {
      loggEvent('Re-fetch', url, { type: type });
    }
  }, [ status ]);

  return result;
}
```

Og på den måten så slipper man å huske å kalle to hooks der man bruker `useFetch`?

Reply:
Sounds good 👍